src/Puzzle.js

```diff
@@ -10,7 +10,7 @@ class Puzzle {
     this.category = puzzleObj['category'];
     this.totalWords = puzzleObj['number_of_words'];
     this.totalLetters = puzzleObj['total_number_of_letters'];
-    this.description = puzzleObj['description'];
+    this.hint = puzzleObj['description'];
```

Comment:
A cool thing you can do to assign a bunch of these instance properties is use destructuring and `Object.assign()` — (look into both of these in your spare time, I posted and es5/es6 video in your channel a while back that goes over them!) Something like this:

```js
let {category, number_of_words, total_number_of_letters } = puzzleObj;
Object.assign(this, {category, number_of_words, total_number_of_letters});
```